ROS2/ROS_workspace/src/cli_app/include/cli.h

```diff
@@ -0,0 +1,12 @@
+#ifndef CLI_H
+#define CLI_H
+
+#include <string>
+#include <vector>
+
+// Function declarations
+void printColoredAsciiImage();
+void handleCommand(const std::string&);
```

Comment:
Does this have to be here since handleCommand is no longer in cli.cpp?

Reply:
It doesn't, I'll fix that!